cypress/support/test-helper.js

```diff
@@ -6,6 +6,8 @@ export const testNonHTMLResponseCode = (path, responseCode) => {
 
 export const getElement = element => cy.get(element);
 
+export const getSecondElement = element => cy.get(element).eq(1);
```

Comment:
@sareh have you considered removing this second function and changing `getElement()` to accept an optional parameter to choose the `nth` element, so this can be more flexible?